scripts/west_commands/runners/pyocd.py

```diff
@@ -158,6 +163,24 @@ def flash(self, **kwargs):
         self.logger.info('Flashing file: {}'.format(fname))
         self.check_call(cmd)
 
+        if self.post_flash_cmd is not []:
```

Comment:
This is not correct usage. In Python, `foo is bar` and `foo is not bar` basically work like pointer comparison in C.

Example:

```py
>>> [] is []
False
```

This should be `if self.post_flash_cmd:` instead, because an empty list is boolean false.